test/integration/testPutGet.js

```diff
@@ -857,3 +861,286 @@ describe('PUT GET test multiple files', function () {
+
+  before(async () => {
+    // Write row data to temp file
+    fs.writeFileSync(tmpFile.name, ROW_DATA);  
+    connection = testUtil.createConnection();
+    await testUtil.connectAsync(connection);
+    await testUtil.executeCmdAsync(connection, createTable);
+  });
+
+  after(async () => {
+    testUtil.deleteFileSyncIgnoringErrors(tmpFile);
+    testUtil.deleteFolderSyncIgnoringErrors(tmpDir);
+
+    await testUtil.executeCmdAsync(connection, removeFile);
+    await testUtil.executeCmdAsync(connection, dropTable);
+    await testUtil.destroyConnectionAsync(connection);
+  });
+
+  const tmpfilePath = getPlatformTmpPath(tmpFile.name);
+  const putQuery = `PUT ${tmpfilePath} ${stage} AUTO_COMPRESS=FALSE`;
+
+  const tmpdirPath = getPlatformTmpPath(tmpDir);
+  const getQuery = `GET ${stage} ${tmpdirPath}`;
+
+  it('testNoCompress', function (done) {
+    async.series(
+      [
+        function (callback) {
+          connection.execute({
+            sqlText: putQuery,
+            complete: function (err, stmt) {
+              if (err) {
+                callback(err);
+              } else {
+                const stream = stmt.streamRows();
+                stream.on('error', function (err) {
+                  callback(err);
+                });
+                stream.on('data', function (row) {
+                  // Check the file is correctly uploaded
+                  assert.strictEqual(row['status'], UPLOADED);
+                  assert.strictEqual(row.targetSize, ROW_DATA_SIZE);
+                });
+                stream.on('end', function () {
+                  callback();
+                });
+              }
+            }
+          });
+        },
+        function (callback) {
+          connection.execute({
+            sqlText: getQuery,
+            complete: function (err, stmt) {
+              if (err) {
+                callback(err);
+              } else {
+                const stream = stmt.streamRows();
+                stream.on('error', function (err) {
+                  callback(err);
+                });
+                stream.on('data', function (row) {
+                  assert.strictEqual(row['status'], DOWNLOADED);
+                  // Verify if the file is not compressed.
+                  const file = path.join(tmpDir, row.file);
+                  const data = fs.readFileSync(file).toString();
+                  assert.strictEqual(data, ROW_DATA);
+                });
+                stream.on('end', function () {
+                  callback();
+                });
+              }
+            }
+          });
+        }
+      ],
+      done
+    );
+  });
+});
+
+describe('PUT GET test with different size', function () {
+  let connection;
+  const stage = `@${DATABASE_NAME}.${SCHEMA_NAME}.%${TEMP_TABLE_NAME}`;
+  const createTable = `create or replace table ${TEMP_TABLE_NAME} (${COL1} STRING, ${COL2} STRING, ${COL3} STRING)`;
+  const removeFile = `REMOVE ${stage}`;
```

Comment:
Better name will be removeFile or removeStage?

Reply:
It is removing files from stage. So technically it is removeFile.